src/librustc_driver/lib.rs

```diff
@@ -632,6 +633,8 @@ Available lint options:
                                     .map(|&(s, _)| s.chars().count())
                                     .max()
                                     .unwrap_or(0);
+    let max_name_len = std::cmp::max(max_name_len, "warnings".len());
```

Comment:
Oh, okay! I should review how importing modules works. I had tried `cmp::max` and that gave me a compiler error, but `std::cmp::max` compiled so I went with that.